api/routes/users.js

```diff
@@ -20,6 +20,31 @@ router.post('/profile', async function(req, res){
 
 });
 
-
+router.post('/create', async function (req, res) {
+    try {
+        // throws if no user
+        await usersData.getUserByEmail(req.googleInfo.email);
+        console.log('User already exists');
+        res.sendStatus(200);
+    } catch {
+        try {
+            const name = req.googleInfo.name.split(' ', 2);
+            // ugly, but prevents errors where a Google user may have a non-standard name
+            const firstName = name[0] || 'First name not provided.';
+            const lastName = name[1] || 'Last name not provided.';
+            await usersData.addUser(
```

Comment:
I see you're using the email also as the display name when adding a user, just wanted to make sure that's what you intended?